### DISCUSSION TITLE
Option for specifying output as unmasked

### DISCUSSION THREAD

**Discussion by krisztian-sala** [community/community#32026] [Actions]
I would really love to have a flag that is the reverse of `add-mask`, like `unmasked`.
I have 2 use cases for this:

1. A third-party action masks an output, but it shouldn't be masked in our use case and we would need to pass it to a different job as well. If it's masked, this is not possible.
2. We have a reusable action that is loading config from a JSON file. We read the config from the same JSON file in our main workflow that is calling the reusable one. If the reusable workflow is called from the same job as which is setting the JSON config as the output, then the output will be masked, because Github decides that it is a secret (it really isn't).

For these cases, the above-mentioned flag would be REALLY useful, because currently, we have to do quite verbose workarounds.
Call it `dangerously-unmask` if you wish, but it would be great to be able to do this.

**Comment by airtower-luna** [comment]
I understand having a value incorrectly masked is annoying, but having a value incorrectly *unmasked* (whether by mistake or malice) could go far beyond annoying into a security incident. Marking that in the name helps only marginally, counting on "people will be careful" has unfortunately been shown not to be a viable security model. Let alone that it wouldn't be obvious if nested in a level of `uses:` or three.
